pallets/crowdloan-reward/src/lib.rs

```diff
@@ -111,56 +98,61 @@ use frame_support::{
     fmt::Debug,
   },
   ensure,
-  Parameter, 
   sp_runtime::traits::{
-    AtLeast32BitUnsigned, 
+    AtLeast32BitUnsigned,
     CheckedSub,
-    MaybeSerializeDeserialize, 
-    Saturating, 
-  }, 
+    Saturating,
+  },
   traits::{
-    Get, 
-    Currency, 
-    ExistenceRequirement::KeepAlive, 
-    EnsureOrigin
-  }, 
+    Currency,
+    EnsureOrigin,
+    ExistenceRequirement::{ KeepAlive, AllowDeath },
+    Get
+  },
   weights::Weight
 };
-
 use frame_system::{
   ensure_root,
   RawOrigin
 };
-
 use sp_runtime::{
   ModuleId,
   Perbill,
   traits::{
     AccountIdConversion,
-    Convert, 
     CheckedDiv,
-    MaybeSerialize, 
-    Member,
-    StaticLookup,    
+    Convert,
+    MaybeSerialize,
+    StaticLookup,
     Zero,
   }
 };
 
+// Re-export in crate namespace (for runtime construction)
+pub use pallet::*;
 // Claim reward trait to be implemented
 use pallet_crowdloan_claim::traits::Reward;
 
 // Extrinsics weight information
 pub use crate::traits::WeightInfo;
 
+// Mock runtime and unit test cases
+mod mock;
```

Comment:
Do you know if the tests and mocks are skipped when building the runtime blob? Should we create its own file outside of lib?

Reply:
The #[cfg(test)] is missing. Then it will be skipped. Will add this to mock and test module.